Sequences/MRML/vtkMRMLNodeSequencer.cxx

```diff
@@ -139,6 +139,17 @@ void vtkMRMLNodeSequencer::NodeSequencer::AddDefaultDisplayNodes(vtkMRMLNode* no
   displayableNode->CreateDefaultDisplayNodes();
 }
 
+void vtkMRMLNodeSequencer::NodeSequencer::AddDefaultSequenceStorageNode(vtkMRMLNode* node)
+{
+  vtkMRMLStorableNode* storableNode = vtkMRMLStorableNode::SafeDownCast(node);
```

Comment:
Is node always a vtkMRMLSequenceNode? If yes, then it would be more clear to use `vtkMRMLSequenceNode* node`.